internal/service/templategenerator/templategenerator.go

```diff
@@ -143,7 +143,7 @@ func (s *Service) GenerateModuleTemplate(
 	labels := generateLabels(moduleConfig)
 	annotations := generateAnnotations(moduleConfig, isCrdClusterScoped)
 
-	ref, err := oci.ParseRef(descriptor.Name)
+	ref, err := oci.ParseRef(moduleConfig.Name)
```

Comment:
I didn't really see why we take it from descriptor instead of the moduleConfig where this is actually originating from. Therefore changed this to use the origin instead.